vtr_flow/benchmarks/noc/Synthetic_Designs/congestion_traffic_flow_files/complex_16_noc_2way_ring.flows

```diff
@@ -0,0 +1,35 @@
+<traffic_flows>
+
```

Comment:
Document what these benchmarks do / how they work / what they can test either in comments in the flow file or in a README that describes a bunch.